dataland-frontend/src/components/resources/dataTable/conversion/MultiLayerDataTableFieldConverter.ts

```diff
@@ -71,3 +78,12 @@ export function getDataModelFieldCellConfig(path: string, field: Field): MLDTCel
     return undefined;
   }
 }
+
+/**
+ * Checks if fields with null values should be shown or not
+ * @param value This is the displayValue parsed from the field config
+ * @returns boolean to set hidden to true or false
+ */
+function shouldValueBeDisplayed(value: MLDTDisplayComponentTypes[MLDTDisplayComponentName]): boolean {
+  return !!(value && value != NO_DATA_PROVIDED);
```

Comment:
the `!!` should not be necessary

Reply:
bekomme sonst einen type assertion error